event_manager/lib/event_manager.rb

```diff
@@ -0,0 +1,79 @@
+require 'csv'
+require 'sunlight/congress'
+require 'erb'
+require 'date'
+
+Sunlight::Congress.api_key = "e179a6973728c4dd3fb1204283aaccb5"
+
+def clean_zipcode(zipcode)
+  zipcode.to_s.rjust(5,"0")[0..4]
+end
+
+def legislators_by_zipcode(zipcode)
+  Sunlight::Congress::Legislator.by_zipcode(zipcode)
+end
+
+def save_thank_you_letters(id,form_letter)
+  Dir.mkdir("output") unless Dir.exists?("output")
+
+  filename = "output/thanks_#{id}.html"
+
+  File.open(filename,'w') do |file|
+    file.puts form_letter
+  end
+end
+
+def validate_phone(phone)
+  if phone.length < 10 
+    return false
+  elsif phone.length == 10 && phone.scan(/\D/).empty?
+    return true
+  elsif phone.length == 11 && phone[0] == "1"
+    phone = phone[1..11]
+  elsif phone.length == 11 && phone[0] != "1"
+    return false
+  elsif phone.length > 11
+    return false
+  else
+    return false
+  end
+end
+
+def get_hour(date)
+  format = "%m/%d/%Y %H:%M"
+  date = DateTime.strptime(date, format)
+  hour = date.to_s.slice(11,14)[0..4]
```

Comment:
it can be written prettier - especially when you already got `DateTime` object.

http://www.rubydoc.info/stdlib/date/DateTime